benchmarks/src/main/java/org/elasticsearch/benchmark/search/fetch/subphase/FetchSourcePhaseBenchmark.java

```diff
@@ -79,13 +79,13 @@ private BytesReference buildBigExample(String extraText) throws IOException {
     @Benchmark
     public BytesReference filterSourceMap() {
         Source bytesSource = Source.fromBytes(sourceBytes);
-        return fetchContext.filter().filterMap(bytesSource).internalSourceRef();
+        return fetchContext.filter(null).filterMap(bytesSource).internalSourceRef();
     }
 
     @Benchmark
     public BytesReference filterSourceBytes() {
         Source bytesSource = Source.fromBytes(sourceBytes);
-        return fetchContext.filter().filterBytes(bytesSource).internalSourceRef();
+        return fetchContext.filter(null).filterBytes(bytesSource).internalSourceRef();
```

Comment:
I think it would be better to add a `filter()` method as well as having a `filter(MappingLookup)` method. It feels weird to me to use a method that mostly is called with `null`.

Reply:
++